pkg/gpu/nvidia/metrics/metrics.go

```diff
@@ -175,53 +171,58 @@ func (m *MetricServer) collectMetrics() {
 	}
 }
 
-func getGpuMetrics(device string, d *nvml.Device) (uint, uint64, error) {
-	status, err := gmc.collectStatus(d)
-	if err != nil {
-		glog.Errorf("Failed to get device status for %s: %v", device, err)
-		return 0, 0, err
+// getGpuMetricsInfo returns dutyCycle, usedMemory, totalMemory, uuid and deviceModel.
+func getGpuMetricsInfo(device string, d *nvml.Device) (uint, uint64, uint64, string, string, error) {
```

Comment:
nit: there are enough return values for this function that a struct might be help the calling code be more clear